utils/posts.js

```diff
@@ -41,15 +43,68 @@ export function getPostMetadata(id) {
             ...matterResult.data,
         },
     };
-
+    
     const title = post.meta.title;
-    var seoTitle = title.toLowerCase().replace(/[^a-z0-9]+/g, ' ').trim().replace(/ /g, '-');
-
+    const seoTitle = title.toLowerCase().replace(/[^a-z0-9]+/g, ' ').trim().replace(/ /g, '-');
+    
     post.meta.seoTitle = seoTitle;
 
+    var tags = post.meta.tags;
+    if(tags) {
+        tags = tags.split(',').map(tag => tag.trim());
```

Comment:
Indentation is off; please run PrettierJS and have it run upon save, so that code is formatted consistently

- https://github.com/prettier/prettier-emacs
- https://github.com/prettier/prettier